service/models.py

```diff
@@ -15,26 +20,36 @@
 class DataValidationError(Exception):
     """ Used for an data validation errors when deserializing """
 
-    pass
-
 
-class YourResourceModel(db.Model):
+class Customer(db.Model):
     """
-    Class that represents a YourResourceModel
+    Class that represents a customers
     """
 
     app = None
 
     # Table Schema
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(63))
+    name = db.Column(db.String(63),nullable=False)
+    lastname = db.Column(db.String(63),nullable=False)
+    email = db.Column(db.String(120))
+    phone = db.Column(db.String(60))
+    address1 = db.Column(db.String(256))
+    address2 = db.Column(db.String(256))
+    city = db.Column(db.String(64))
+    state = db.Column(db.String(16))
+    country = db.Column(db.String(63))
+    zipcode = db.Column(db.String(60))
+    created_at = db.Column(db.DateTime(timezone=True), server_default=func.now())
```

Comment:
Why are you using func.now() here and datetime.now in the next line, is there a difference to the formatting?